src/Solver/Dynamic/Ida.cpp

```diff
@@ -197,6 +189,7 @@ namespace AnalysisManager
         if (tag_)
           initType = IDA_YA_YDP_INIT;
 
+        // TODO: this doesn't doing anything without calling IDAGetConsistentIC
         retval = IDACalcIC(solver_, initType, 0.1);
         checkOutput(retval, "IDACalcIC");
```

Comment:
I am not sure I understand this comment. My understanding is `IDACalcIC` overwrites solution vectors `yy_` and `yp_`.

Reply:
I was thinking the consistent initial condition would need to be copied back into the model, but I realize now that's probably not necessary.